3rdparty/pcrs/pcrs-0.0.3/pcrs.3.in

```diff
@@ -0,0 +1,488 @@
+.\" Copyright (c) 2001-2003 Andreas S. Oesterhelt <oes@oesterhelt.org>
+.\"
+.\" This is free documentation; you can redistribute it and/or
+.\" modify it under the terms of the GNU General Public License as
+.\" published by the Free Software Foundation; either version 2 of
```

Comment:
It can't be imported in our tree. 

First it's GPLv2, which is incompatible to GPLv3. Second, we can't ship 3rd party GPL code in our binaries, even if they are GPLv3. Copyright must be CFEngine AS.

I'd suggest you implement linking to it as a third party library, for example: `./configure --with-pcrs=/usr/local`